packages/components/src/globals/scss/vendor/@carbon/themes/scss/_tokens.scss

```
@@ -313,3 +313,7 @@ $active-01: map-get($carbon--theme, 'active-01') !default;
 /// @alias hover-ui
 /// @deprecated
 $hover-field: map-get($carbon--theme, 'hover-field') !default;
+/// @type Color
+/// @access public
+/// #group @carbon/themes
```

```suggestion
/// @group @carbon/themes
```

Does it need to be like this?